src/client/sandbox/event/simulator.js

```diff
@@ -538,17 +538,30 @@ export default class EventSimulator {
     }
 
     _dispatchEvent (el, name, shouldBubble, flag) {
-        var ev = null;
+        var event                     = null;
```

Comment:
@helen-dikareva Do you like the idea to split this into two functions `_dispatchEvent` and `_dispatchFocusEvent` ? On my opinion it would be better